src/apps/packet_filter/pcap_filter.lua

```diff
@@ -65,13 +67,15 @@ local basic_apps = require("apps.basic.basic_apps")
 function selftest ()
    print("selftest: pcap_filter")
    selftest_run(false, 3.726, 0.0009)
-   selftest_run(true,  3.800, 0.1)
+   selftest_run(true,  7.453, 0.001)
```

Comment:
Of the 161 packets in the sample, 6 are passed by the filter, that makes 3.7267% for the non-stateful test.
Each one of these packets has an answer, so the stateful filter passes 12 packets (in fact 11, since one of these appears before the original, but the the sample is repeated many times, so it approaches 12/161)
The original test didn't clear the app counters, so the reported rate was in fact `(6*n0+12*n1)/(161*(n0+n1))`, where `n0` is the number of repetitions during the first (non-stateful) phase and `n1` during the stateful phase.

That had me very confused for a few hours; i was getting nonsense rates (in the order of 5%), and it seemed very performance-sensitive.  specially when i inserted some `io.write(...)` to test, it passed the test!  I thought i was hitting some JIT dependency or bug, but when i inserted the `app.configure(config.new())` line to clear stats, the rates stabilized at 7.4534%

(from this we can deduce the stateful test did _a lot_ less repetitions than the non-stateful, in the order of 50 times less.  the new version does roughly half as many of the non-stateful)